packages/core/src/base-chart.ts

```diff
@@ -951,4 +953,14 @@ export class BaseChart {
 	getBBox(selector: any) {
 		return this.innerWrap.select(selector).node().getBBox();
 	}
+
+	// ================================================================================
+	// Makes chart data components more screen reader friendly
+	// ================================================================================
+	makeAccessible (innerWrap: any, label?: any) {
+		// Make chart data components tabbable
+		innerWrap.attr("tabindex", 0)
```

Comment:
innerWrap should not be passed into `makeAccesible`. Can be accessed using `this.innerWrap`.

Reply:
I don't have to pass innerWrap in, but leaving innerWrap?:any as a parameter does the trick